src/teleop/jetson/jetson_teleop.py

```diff
@@ -97,37 +100,108 @@ def __init__(self):
+
+        # Filter for xbox triggers, they are typically [-1,1]
         raw_left_trigger = msg.axes[self.xbox_mappings["left_trigger"]]
         left_trigger = raw_left_trigger if raw_left_trigger > 0 else 0
         raw_right_trigger = msg.axes[self.xbox_mappings["right_trigger"]]
         right_trigger = raw_right_trigger if raw_right_trigger > 0 else 0
+
         self.ra_cmd.velocity = [
-            self.ra_config["joint_a"]["multiplier"]
-            * quadratic(deadzone(msg.axes[self.xbox_mappings["left_js_x"]], 0.15)),
-            self.ra_config["joint_b"]["multiplier"]
-            * quadratic(-deadzone(msg.axes[self.xbox_mappings["left_js_y"]], 0.15)),
-            self.ra_config["joint_c"]["multiplier"]
-            * quadratic(-deadzone(msg.axes[self.xbox_mappings["right_js_y"]], 0.15)),
-            self.ra_config["joint_d"]["multiplier"]
-            * quadratic(deadzone(msg.axes[self.xbox_mappings["right_js_x"]], 0.15)),
+            self.ra_config["joint_a"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_x", 0.15, True),
+            self.ra_config["joint_b"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_y", 0.15, True),
+            self.ra_config["joint_c"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_y", 0.15, True),
+            self.ra_config["joint_d"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_x", 0.15, True),
             self.ra_config["joint_e"]["multiplier"] * (right_trigger - left_trigger),
             self.ra_config["joint_f"]["multiplier"]
-            * (msg.buttons[self.xbox_mappings["right_bumper"]] - msg.buttons[self.xbox_mappings["left_bumper"]]),
-            self.ra_config["finger"]["multiplier"]
-            * (msg.buttons[self.xbox_mappings["y"]] - msg.buttons[self.xbox_mappings["a"]]),
-            self.ra_config["gripper"]["multiplier"]
-            * (msg.buttons[self.xbox_mappings["b"]] - msg.buttons[self.xbox_mappings["x"]]),
+            * self.filter_xbox_button(msg.buttons, "right_bumper", "left_bumper"),
+            self.ra_config["finger"]["multiplier"] * self.filter_xbox_button(msg.buttons, "y", "a"),
+            self.ra_config["gripper"]["multiplier"] * self.filter_xbox_button(msg.buttons, "b", "x"),
         ]
         self.ra_cmd_pub.publish(self.ra_cmd)
 
+    def sa_control_callback(self, msg: Joy) -> NoReturn:
+        """
+        Converts a Joy message with the Xbox inputs
+        to a JointState to control the RA Arm in open loop
+        :param msg: Has axis and buttons array for Xbox controller
+        Velocities are sent in range [-1,1]
+        :return:
+        """
+
+        # Filter for xbox triggers, they are typically [-1,1]
+        raw_left_trigger = msg.axes[self.xbox_mappings["left_trigger"]]
+        left_trigger = raw_left_trigger if raw_left_trigger > 0 else 0
+        raw_right_trigger = msg.axes[self.xbox_mappings["right_trigger"]]
+        right_trigger = raw_right_trigger if raw_right_trigger > 0 else 0
```

Comment:
Wait how does this work? Does this mean we're losing half the input space?

Reply:
We are losing half the input but not the output. IIRC @tabiosg had said that this would be fine in his ex-driver opinion. This could also just be swapped for fixing the input range to be [0,1] for each trigger.

Reply:
Deferring to a separate issue, good enough to merge for now